test/parallel/test-stream-pipeline.js

```diff
@@ -86,6 +86,54 @@ const net = require('net');
   }));
 }
 
+{
+	const w = new Writable({
+	  write (chunk, enc, cb) {
+	    cb()
+	  }
+	});
+
+	function createTransformStream (tf, context) {
+	  return new Transform({
+	    readableObjectMode: true,
+	    writableObjectMode: true,
+
+	    transform (chunk, encoding, done) {
+	      tf(chunk, context, done);
+	    }
+	  });
+	}
+
+	const ts = createTransformStream((chunk, _, done) => done(new Error('Artificial error')));
+
+	pipeline(ts, w, common.mustCall((err) => {
+		assert.ok(err, 'should have an error');
+	}))
+
+	ts.write('test')
+}
+
+{
+	function createTransformStream (tf, context) {
+	  return new Transform({
+	    readableObjectMode: true,
+	    writableObjectMode: true,
+
+	    transform (chunk, encoding, done) {
+	      tf(chunk, context, done);
+	    }
+	  })
+	}
+
+	const ts = createTransformStream((chunk, _, done) => done(new Error('Artificial error')));
+
+	pipeline(ts, process.stdout, common.mustCall((err) => {
```

Comment:
This is not correct. You should test this in the `test/pseudo-tty` folder.

Reply:
I'm working on the test mentioned